w2w/resources/.~lock.LCZ_UCP_lookup.csv#

```diff
@@ -0,0 +1 @@
+Matthias Demuzere,demuzmp4,rub.geographie.ruhr-uni-bochum.de,14.01.2022 14:47,file:///home/demuzmp4/.config/libreoffice/4;
```

Comment:
Was this checked in by accident? Let me know, then I can easily remove it without a separate commit.

Reply:
Ha, yes. Libreoffice must have been open when committing. This file can be removed.